app/kubemci/pkg/validations/validations.go

```diff
@@ -89,3 +91,69 @@ func nodePortSameInAllClusters(backend v1beta1.IngressBackend, namespace string,
 	}
 	return nil
 }
+
+func ServerVersionsNewEnough(clients map[string]kubeclient.Interface) (bool, error) {
+	for key := range clients {
+		glog.Infof("Checking client %s", key)
+		discoveryClient := clients[key].Discovery()
+		if discoveryClient == nil {
+			return false, fmt.Errorf("no discovery client in client: %+v", clients[key])
+		}
+		ver, err := discoveryClient.ServerVersion()
+		if err != nil {
+			return false, fmt.Errorf("could not get discovery client to lookup server version: %s", err)
+		}
+		glog.Infof("ServerVersion: %+v", ver)
+		newEnough, err := serverVersionNewEnough(ver.GitVersion)
+		if err != nil {
+			return false, fmt.Errorf("error checking server version: %s", err)
+		}
+		if !newEnough {
+			fmt.Println("Cluster", key, "is not running a supported kubernetes version. Need at least 1.8.1.")
+			return false, nil
+		}
+
+	}
+	return true, nil
+}
+
+func serverVersionNewEnough(version string) (bool, error) {
+	// Example string we're matching: v1.9.3-gke.0
```

Comment:
FYI thats for GKE clusters. Non GKE clusters will return "v1.9.3", without the gke suffix. We should test both in unit test

Reply:
most of the tests are without the gke suffix, but i've now added one with "-gke.0"